spec/system/admin/order_cycles/edit_spec.rb

```diff
@@ -157,9 +157,9 @@
         expect(page).to have_content('You have unsaved changes')
 
         click_button('Save')
+        expect(page).to have_content('Your order cycle has been updated.')
         expect(page).not_to have_content "Orders are linked to this cycle"
         expect(page).to have_field 'order_cycle_orders_close_at', with: '2024-03-30 00:00'
-        expect(page).to have_content('Your order cycle has been updated.')
```

Comment:
I think you got it right 💪 : the 'not_to have_content' assertion could have passed straight away. 

What your change now says is "_After_ changes have been saved, expect page not to have content "orders are linked".